addon/ng2/tasks/uninstall.js

```diff
@@ -0,0 +1,157 @@
+/* jshint node: true, esnext: true */
+'use strict';
+
+const Promise     = require('ember-cli/lib/ext/promise');
+const Task        = require('ember-cli/lib/models/task');
+const chalk       = require('chalk');
+const path        = require('path');
+const fs          = require('fs');
+const fse         = require('fs-extra');
+const exec        = Promise.denodeify(require('child_process').exec);
+const packageJSON = path.resolve(process.cwd(), 'package.json');
+const nodeModules = path.resolve(process.cwd(), 'node_modules');
+
+module.exports = Task.extend({
+  command: '',
+  completionOKMessage: 'Successfully uninstalled.',
+  completionErrorMessage: 'Error uninstalling package.',
+
+  run: function(options) {
+    this.package = options.package;
+    this.typings = options.typings;
+
+    this.disableLogger();
```

Comment:
This is indeed needed, I think. `npm install` otherwise outputs some info about the process that we don't want. I actually took that from one existing `ember-cli` task, but not sure which.